tools/tokens/daoTxns.js

```diff
@@ -17,10 +17,29 @@ var DAOCreatedToken = mongoose.model('DAOCreatedToken');
 var DAOTransferToken = mongoose.model('DAOTransferToken');
 var InternalTx     = mongoose.model( 'InternalTransaction' );
 
+// load config.json
+var config = { nodeAddr: 'localhost', gethPort: 8545 };
```

Comment:
Thanks for your code contribution. I will approve it, but wondering if you can refactor it first. It seems to me that it's the same piece of code added to all those different files. A more optimal way to do it is to have on file with that piece of logic that you import into the required files needed.

So, say you have this file here:
```
var config = { nodeAddr: 'localhost', gethPort: 8545 };
try {
    var local = require('../../config.json');
    _.extend(config, local);
    console.log('config.json found.');
} catch (error) {
    if (error.code === 'MODULE_NOT_FOUND') {
        var local = require('../../config.example.json');
        _.extend(config, local);
        console.log('No config file found. Using default configuration... (config.example.json)');
    } else {
        throw error;
        process.exit(1);
    }
}
```
All you need to do is just import the `config` from that separate file into all the other files. It's also more beneficial as you don't need to have different references to your `config.json` path like here: `var local = require('../../config.example.json');`

Let me know if you have any questions. It's the more engineering approach, trying to abstract everything whenever you can. It's less code to read that way and becomes a more optimal code base. I'll be making comments over where I think you can abstract the code.